### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by kennedyowusu**
## Highlights

- [ ] Add styles to the project
- [ ] Ensure all linters pass

**Below is a screenshot of the outcome after adding styles**

book-store-style

### CODE REVIEW

File: src/components/Books/InputForm.jsx [hunk 1]
```diff
@@ -37,26 +37,45 @@ const InputForm = () => {
...
-                Please fill in all fields
-              </p>
-            )
-            : null
-        }
+        <button type="submit" className="primary-button-big">
+          Add Book
+        </button>
+        {validateForm ? (
+          <p className="error" style={{ color: 'red' }}>
```
Comment (rustam-tolipov) [inline src/components/Books/InputForm.jsx]: - [ ] This inline styling is not appropriate for this project. Kindly move this styling into an external CSS file. For more info kindly follow this link: [[Good and Bad CSS]](https://speckyboy.com/good-bad-css-practices/#:~:text=your%20site%20optimized.-,Using%20Inline%20Styles,-Using%20Inline%20Styles)

File: public/index.html [hunk 2]
```diff
@@ -10,34 +10,14 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```
Comment (rustam-tolipov) [inline public/index.html:12]: - [ ] In your readme file, it seems like your MIT license link is broken. Kindly connect your MIT license with the correct link. Thank you!

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @kennedyowusu,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books/InputForm.jsx (hunk 1 above)
Comment (kennedyowusu) [reply src/components/Books/InputForm.jsx]: Thank you for the feedback. This issue has been resolved now

File: public/index.html (hunk 2 above)
Comment (kennedyowusu) [reply public/index.html:12]: Thank you for your feedback. I have resolved this issue. 

[MIT LICENSE](https://github.com/kennedyowusu/bookstore/blob/develop/LICENSE.md)

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @kennedyowusu 

Your project is complete! Great job for making the requested changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request✔️ 
- Style well implemented✔️ 
- Readme ✔️ 
- No linter errors✔️ 

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**